From 23ce7f8c727d76d4532b8949c383c3f17b64a539 Mon Sep 17 00:00:00 2001
From: Marco Genasci <fedeliallalinea@gmail.com>
Date: Fri, 20 Apr 2018 11:40:30 +0200
Subject: [PATCH] libzipplugin: Add missing <memory> header

Summary:
Build fails on some setups for use of std::unique_ptr.
See also: https://bugs.gentoo.org/653546

Reviewers: #ark

Subscribers: kde-utils-devel

Tags: #ark

Differential Revision: https://phabricator.kde.org/D12374
---
 plugins/libzipplugin/libzipplugin.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plugins/libzipplugin/libzipplugin.cpp b/plugins/libzipplugin/libzipplugin.cpp
index 5641c97b..be260b39 100644
--- a/plugins/libzipplugin/libzipplugin.cpp
+++ b/plugins/libzipplugin/libzipplugin.cpp
@@ -41,6 +41,7 @@
 
 #include <utime.h>
 #include <zlib.h>
+#include <memory>
 
 K_PLUGIN_FACTORY_WITH_JSON(LibZipPluginFactory, "kerfuffle_libzip.json", registerPlugin<LibzipPlugin>();)
 
-- 
2.17.0

